packages/c/sshnpd/src/handle_npt_request.c

```diff
@@ -483,8 +507,19 @@ void handle_npt_request(atclient *atclient, pthread_mutex_t *atclient_lock, sshn
       free(session_iv_base64);
     }
 
-    int res = run_srv_process(params, rvd_host, rvd_port, true, requested_host, requested_port, authenticate_to_rvd,
-                              rvd_auth_string, encrypt_rvd_traffic, true, session_aes_key, session_iv);
+    char *rvd_host_str = cJSON_GetStringValue(rvd_host);
+    uint16_t rvd_port_int = cJSON_GetNumberValue(rvd_port);
+
+    char *requested_host_str = cJSON_GetStringValue(requested_host);
+    uint16_t requested_port_int = cJSON_GetNumberValue(requested_port);
+
+    const bool multi = true;
+
+    int res = run_srv_process(rvd_host_str, rvd_port_int, requested_host_str, requested_port_int, authenticate_to_rvd,
+                              rvd_auth_string, encrypt_rvd_traffic, multi, session_aes_key, session_iv);
+    free(rvd_host_str);
+    free(requested_host_str);
+
```

Comment:
run_srv_process signature change uptake, we now convert the cJSON references first, instead of passing them in.